dali/python/nvidia/dali/pipeline.py

```diff
@@ -116,13 +118,29 @@ class Pipeline(object):
 `enable_memory_stats`: bool, optional, default = False
     If DALI should print operator output buffer statistics.
     Usefull for `bytes_per_sample_hint` operator parameter.
+`py_workers_num`: int, optional, default = None
```

Comment:
trailing `num` is not good - `num_workers` or `worker_count`
"worker number" - primary meaning is a "worker ID" rather than number of workers
"workers number" - not English
Googling:
num_workers - 990,000 results
worker_num - 35,000 results
workers_num - 14,000 results